package.json

```diff
@@ -51,6 +51,7 @@
   },
   "ember-addon": {
     "configPath": "tests/dummy/config",
-    "demoURL": "https://miguelcobain.github.io/ember-fullscreen/"
+    "demoURL": "https://miguelcobain.github.io/ember-fullscreen/",
+    "after": "ember-cli-shims"
```

Comment:
Why is this line needed?

Reply:
It is not needed. I added that line in a previous commit to make a test, but I removed it.